### DISCUSSION TITLE
Is Redis for acceleration or as the backend hosting the vulnerability database

### DISCUSSION THREAD

**Discussion by Vad1mo** [aquasecurity/trivy#1214] [Q&A]
This is not clear to me from the documentation. I want to run Trivy in server mode with a read only filesystem. 

There is the option to [use REDIS](https://aquasecurity.github.io/trivy/v0.18.3/examples/cache/) for caching. 

It's not clear to me whether REDIS is there only for caching and the db is still on FS or if Redis can hold the complete vulnerability database so we can run fully stateless in a read-only container.

**Comment by catbabygabby** [comment]
hi

**Reply by catbabygabby** [reply]
are you guys text to me please people all like girl
